Automattic/wp-calypso
# Update stripe setup controls

#### Changes proposed in this Pull Request

Updated the Stripe Connect dialog from segmented tabs to radio buttons. 

#### Testing instructions



Switch to this branch. 
Visit https://wordpress.com/store/settings/payments/ 
Click Set Up. 
Verify functionality and visuals. 

![Screen Shot 2019-08-06 at 1 06 25 PM](https://user-images.githubusercontent.com/181733/62734873-d80b2c80-b9de-11e9-82ad-b3cb8c562905.png)

Fixes #33271
